### ISSUE TITLE
Budget proposal not being paid

### ISSUE THREAD

**Issue by RealHoneyBadger** [LunariumCoin/lunarium#26]
Below test proposal failed to pay out leading me to believe that there must be a bug in the code.
```
    "Name": "testing",
    "URL": "https://discord.gg/4nFZeJr",
    "Hash": "7321a7bc082f933f418563af872a21b9b5707ca386de33866dd36e486faaa49d",
    "FeeHash": "fda9586854d1665e71f3783508b984299d096f2fe91c99654fe2a816675d5310",
    "BlockStart": 43200,
    "BlockEnd": 86401,
    "TotalPaymentCount": 1,
    "RemainingPaymentCount": 0,
    "PaymentAddress": "LSZSmLaHy4D6M7PU9zhb7iBunGRRkauyqU",
    "Ratio": 1,
    "Yeas": 32,
    "Nays": 0,
    "Abstains": 0,
    "TotalPayment": 50.00000000,
    "MonthlyPayment": 50.00000000,
    "IsEstablished": true,
    "IsValid": true,
    "IsValidReason": "",
    "fValid": true
```

**Comment by jackkdev** [comment]
Believe this was fixed and resolved at block 400K?

**Comment by RealHoneyBadger** [comment]
Closing issue. Fixed in version 1.2.0
